development/tools/cmd/vulnerabilitycollector/README.md

```diff
@@ -0,0 +1,25 @@
+# Vulnerability Collector
+
+## Overview
+The tool queries the public google container repository API for all containers it can find for a given project, while traversing paths as they're found. With the generated list of containers, it will pick the last built container and runs a check for any security vulnerabilities found from the GCR.io vulnerability scanner. If a region or path is specified, the number of queries is drastically reduced.
```

Comment:
The overview does not really say what the Vulnerability Collector is and what it does. Please rewrite the section. Is its only function to query the public Google container repository API?

Reply:
As of now, this is the only thing we do with it, yes.

Reply:
```suggestion
The Vulnerability Collector queries the public Google container repository API about all containers in a given project while traversing paths as they're found. The result is a generated list of containers. The Vulnerability Collector picks the latest container and runs a check for any security vulnerabilities found by the GCR.io vulnerability scanner. If a region or path is specified, the number of queries is drastically reduced.
```